package.json

```diff
@@ -8,7 +8,7 @@
         "dev": "nodemon index.js",
         "prepare": "husky install",
         "lint-staged": "lint-staged",
-        "test:bdd": "cucumber-js --fail-fast --format progress --format-options '{\"colorsEnabled\": true}' test/bdd/ -r test/bdd/steps/",
+        "test:bdd": "cucumber-js --publish --fail-fast --format progress --format-options '{\"colorsEnabled\": true}' test/bdd/ -r test/bdd/steps/",
```

Comment:
--publish is not needed you can remove it

Reply:
Milos told that is useful to include the Cucumber report, this --publish tag generates the report. I will remove it.